src/components-v2/vault-detail/actions/MobileStickyActionButtons.tsx

```diff
@@ -29,35 +34,50 @@ interface Props {
 
 export const MobileStickyActionButtons = observer(({ vault, onDepositClick, onWithdrawClick }: Props): JSX.Element => {
   const classes = useStyles();
-  const { vaults, wallet } = React.useContext(StoreContext);
+  const { vaults, wallet, user, chain: networkStore } = React.useContext(StoreContext);
   const canUserDeposit = wallet.isConnected ? vaults.canUserDeposit(vault) : false;
   const canUserWithdraw = vaults.canUserWithdraw(vault);
+  const { depositBalance } = useVaultInformation(vault);
+  const isUserHasToken = user.getBalance(vault.underlyingToken).hasBalance();
+  const isUserHasDeposit = !depositBalance.tokenBalance.eq(0);
+  const { network } = networkStore;
+  const strategy = Chain.getChain(network).strategies[vault.vaultToken];
 
   return (
     <div className={classes.root}>
       <Grid container spacing={1}>
         <Grid item xs>
-          <VaultActionButton
-            fullWidth
-            color="primary"
-            variant={canUserDeposit ? 'contained' : 'outlined'}
-            disabled={!canUserDeposit}
-            onClick={onDepositClick}
-          >
-            Deposit
-          </VaultActionButton>
-        </Grid>
-        <Grid item xs>
-          <VaultActionButton
-            color="primary"
-            variant="outlined"
-            fullWidth
-            disabled={!canUserWithdraw}
-            onClick={onWithdrawClick}
-          >
-            Withdraw
-          </VaultActionButton>
+          {isUserHasToken ? (
+            <VaultActionButton
+              fullWidth
+              color="primary"
+              variant={canUserDeposit ? 'contained' : 'outlined'}
+              disabled={!canUserDeposit}
+              onClick={onDepositClick}
+            >
+              Deposit
+            </VaultActionButton>
+          ) : (
+            <Link href={strategy.depositLink} target="_blank" className={classes.goToLink} underline="none">
+              <VaultActionButton variant="contained" fullWidth color="primary">
+                Go to {vault.protocol}
```

Comment:
for this one we will need some reappings 

- aura -> balancer
- convex -> curve

as those deposits will be on the underlying protocol now the one the vault farms on

Reply:
@axejintao do I need to show `Go to Balancer` instead of `Go to Aura` and `Go to Curve` instead of `Go to Convex`?

Reply:
correct